src/wp-admin/about.php

```diff
@@ -88,16 +84,16 @@
 					<?php _e( 'Block themes now include the option to contain multiple style variations. This expands the new Style system even further and enables shortcuts to switch the look and feel of your site all within a single theme. You can change both the available settings, like the font weight, and the style options, like the default color palette. Change the look and feel of your site with just a few clicks.' ); ?>
 				</p>
 			</div>
-			<div class="column about__image is-vertically-aligned-center">
+			<div class="column about__image is-vertically-aligned-top">
 				<img src="https://make.wordpress.org/core/files/2022/05/about-60-style-switcher.gif" alt="" />
 			</div>
 		</div>
 
 		<div class="about__section has-1-column">
-			<div class="column about__image is-vertically-aligned-center">
+			<div class="column about__image is-vertically-aligned-top">
 				<img src="https://make.wordpress.org/core/files/2022/05/about-60-templates.png" alt="" />
 			</div>
-			<div class="column">
+			<div class="column" style="padding-bottom:0">
```

Comment:
Given this is used a few times, it it possible to add an `is-padding...` helper class to the admin css.

Nit, not a blocker.